examples/mvc/controllers/user/views/list.pug

```diff
@@ -4,4 +4,4 @@ p Click a user below to view their pets.
 ul
   each user in users
     li
-      a(href='/user/#{user.id}')= user.name
+      a(href='/user/' + user.id)= user.name
```

Comment:
Node 4+ would allow this to be written as `a(href=`/user/${user.id}`)= user.name`